tb_plugin/README.md

```diff
@@ -258,11 +272,76 @@ The suspended toolbar has functionalities to help view the trace line.
 For example, when the up-down arrow is enabled,
 you can zoom in by dragging the mouse up and keeping mouse's left button pushed down.
 
-![Alt text](https://github.com/pytorch/kineto/blob/master/tb_plugin/docs/images/trace_view_one_step.PNG)
+![Alt text](https://github.com/guyang3532/kineto/blob/readme/tb_plugin/docs/images/trace_view_one_step.PNG)
 
 The “Optimizer.step#SGD.step” and ”enumerate(DataLoader)#_SingleProcessDataLoaderIter.\__next\__”
 are high-level python side functions.
 
 When you select the top-right corner's “Flow events” to ”async”,
 you can see the relationship between an operator and its launched kernels.
-![Alt text](https://github.com/pytorch/kineto/blob/master/tb_plugin/docs/images/trace_view_launch.PNG)
+![Alt text](https://github.com/guyang3532/kineto/blob/readme/tb_plugin/docs/images/trace_view_launch.PNG)
+
+You can also view the gpu utilization and Est. SM Efficiency in the trace view. They are drawn alongside the timeline:
+
+![Alt text](https://github.com/guyang3532/kineto/blob/readme/tb_plugin/docs/images/trace_view_gpu_utilization.PNG)
+
+* Memory View
+
+    Pytorch profiler records all memory allocation/release events during profiling. For each operator, the plugin aggregates all the events
+    inside its life span.
+
+    ![Alt text](https://github.com/guyang3532/kineto/blob/readme/tb_plugin/docs/images/memory_view.PNG)
```

Comment:
Using "GPU0" as picture is better than "CPU"?

Reply:
done, thanks